log-analyzer/features/org.wso2.carbon.la.apim.feature/src/main/resources/gadgets/LogAPIMArtifactDeleted/js/main.js

```diff
@@ -0,0 +1,144 @@
+/*
+ * Copyright (c)  2016, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+var client = new AnalyticsClient().init();
+var div = "#tblArtifactDeleted";
+var table;
+var from = new Date(moment().subtract(1, 'year')).getTime();
+var to = new Date(moment()).getTime();
+var dataM = [];
+var initState = true;
+
+var meta = {
+    "names": ["apiArtifact", "Frequency"],
+    "types": ["ordinal", "linear"]
+};
+
+var configTable = {
+    key: "apiArtifact",
+    title:"LogAPIMArtifact",
+    charts: [{
+        type: "table",
+        y: "Frequency",
+        columns: ["apiArtifact", "Frequency"],
+        columnTitles: ["APIM Artifact", "Frequency"]
+    }
+    ],
+    width: $('body').width(),
+    height: $('body').height(),
+    padding: { "top": 40, "left": 80, "bottom": 70, "right": 100 }
+};
+
+
+function initialize() {
+    fetch();
+    //$("#tblArtifactDeleted").html(getDefaultText());
+}
+
+function getDefaultText() {
+    return '<div class="status-message">'+
+        '<div class="message message-info">'+
+        '<h4><i class="icon fw fw-info"></i>No content to display</h4>'+
+        '<p>Please select a date range to view stats.</p>'+
+        '</div>'+
+        '</div>';
+};
+
+function getEmptyRecordsText() {
+    return '<div class="status-message">'+
+        '<div class="message message-info">'+
+        '<h4><i class="icon fw fw-info"></i>No records found</h4>'+
+        '<p>Please select a date range to view stats.</p>'+
+        '</div>'+
+        '</div>';
+}
+
+$(document).ready(function () {
+    initialize();
+});
+
+function fetch() {
+    dataM.length = 0;
+    var queryInfo;
+    console.log("sajith3");
```

Comment:
Please remove this log, which doesn't have any meaningful information than your name..